tests/Functional/Schema/SQLServerSchemaManagerTest.php

```diff
@@ -189,152 +139,65 @@ public function testColumnComments(): void
         self::assertEquals('Doctrine array type.', $columns['commented_type_with_comment']->getComment());
         self::assertEquals('Some comment', $columns['commented_req_change_column']->getComment());
 
-        $tableDiff            = new TableDiff('sqlsrv_column_comment');
-        $tableDiff->fromTable = $table;
-
-        $tableDiff->addedColumns['added_comment_none']
-            = new Column('added_comment_none', Type::getType('integer'));
-
-        $tableDiff->addedColumns['added_comment_null']
-            = new Column('added_comment_null', Type::getType('integer'), ['comment' => null]);
-
-        $tableDiff->addedColumns['added_comment_false']
-            = new Column('added_comment_false', Type::getType('integer'), ['comment' => false]);
-
-        $tableDiff->addedColumns['added_comment_empty_string']
-            = new Column('added_comment_empty_string', Type::getType('integer'), ['comment' => '']);
-
-        $tableDiff->addedColumns['added_comment_integer_0']
-            = new Column('added_comment_integer_0', Type::getType('integer'), ['comment' => 0]);
-
-        $tableDiff->addedColumns['added_comment_float_0']
-            = new Column('added_comment_float_0', Type::getType('integer'), ['comment' => 0.0]);
-
-        $tableDiff->addedColumns['added_comment_string_0']
-            = new Column('added_comment_string_0', Type::getType('integer'), ['comment' => '0']);
-
-        $tableDiff->addedColumns['added_comment']
-            = new Column('added_comment', Type::getType('integer'), ['comment' => 'Doctrine']);
-
-        $tableDiff->addedColumns['`added_comment_quoted`']
-            = new Column('`added_comment_quoted`', Type::getType('integer'), ['comment' => 'rulez']);
-
-        $tableDiff->addedColumns['select']
-            = new Column('select', Type::getType('integer'), ['comment' => '666']);
-
-        $tableDiff->addedColumns['added_commented_type']
-            = new Column('added_commented_type', Type::getType('object'));
-
-        $tableDiff->addedColumns['added_commented_type_with_comment']
-            = new Column('added_commented_type_with_comment', Type::getType('array'), ['comment' => '666']);
-
-        $tableDiff->renamedColumns['comment_float_0']
-            = new Column('comment_double_0', Type::getType('decimal'), ['comment' => 'Double for real!']);
```

Comment:
The comment in the renamed column diff (`'Double for real!'`) cannot be different from the original column comment (`'0'`) because the comparator would not detect this as a rename.